src/Blocks/Range.purs

```diff
@@ -0,0 +1,37 @@
+module CN.UI.Block.Range (range) where
+
+import Prelude
+
+import DOM.HTML.Indexed (HTMLinput)
+import Halogen.HTML as HH
+import Halogen.HTML.Properties as HP
+
+range
+  :: ∀ p i
+   . String
+  -> Number
+  -> Number
+  -> String
+  -> String
+  -> Array (HH.IProp HTMLinput i)
+  -> HH.HTML p i
+range inputId min max minLabel maxLabel iprops =
+  HH.div
+    [ HP.class_ $ HH.ClassName "flex items-center w-full" ]
+    [ HH.span
+      [ HP.class_ $ HH.ClassName "flex-no-grow text-sm mr-4" ]
+      [ HH.text minLabel ]
+    , HH.input
```

Comment:
I'd love to figure out some standard way to decide how to pass things in like this. For example, in the `Input` block, you just pass in props, and so it's up to you to include an id in your props if you need one. Here, these are required explicitly. We'll need to settle on some standard style for this. @foresttoney @whoadave

Reply:
Yeah, I'm a bit torn. On the one hand I like keeping things consistent and passing arrays of props, so that our blocks work in practically the same way as the standard element functions (however this particular example will at the very least have the labels passed in outside of any props).

On the other hand, if we know that every input is going to have an id, and every range is going to have a min and max, it seems reasonable to abstract some of that boilerplate.

I could go either way.

Reply:
The boilerplate here is just between passing `[ HP.id_ "my-id", HP.min 30.0, HH.class_ ... ]` vs. `func "my-id" 30.0 [ HH.class_ ... ]` though, and so I'm not sure the boilerplate gain is significant. 

Having to supply arguments carries a larger cognitive load than just passing in iprops the way you would anywhere else, and that makes me lean towards straight iprops -- the same as the 'principle of least surprise' Dan was talking about last week (it works this way here, so I expect this similar situation to work the same way).

...but on the other hand, the extra cognitive load of the arguments may be a good thing because it means you actually think about them, and don't forget to supply an `id_` or something. Arguments make this a lot more explicit.